test.mjs

```diff
@@ -251,6 +251,20 @@ if (test('Retry works')) {
   assert(Date.now() >= now + 50 * (5 - 1))
 }
 
+if (test('spawn timeout')) {
+  let signal = 0
+  $.spawn.timeout = 100
+  $.spawn.killSignal = 'SIGKILL'
+  try {
+    await $`sleep 9999`
+  } catch (p) {
+    signal = p.signal
+  }
+  delete $.spawn.timeout
+  delete $.spawn.killSignal
+  assert.equal(signal, 'SIGKILL')
+}
+
```

Comment:
Probably we can introduce another experimental func (as retry):
```js
await cutoff('SIGKILL', 1000)`...`
```

Reply:
What's about `$.o({...})` factory?

```ts
$.nothrow = $.o({noThrow: true})
$.quiet = $.o({quiet: true})
$.retry = $.o({retry: 5, delay: 1000})
...
$.custom = $.o({timeout: 10000, maxBuffer: 1024 * 1024 * 1024, retry: 5, delay: 1000, quiet: true})
```

Reply:
Or maybe introduce `$.options`?

```ts
Object.assign($.options, {
    quiet: true,
    retry: 5,
    delay: 1000
})

$`cmd args`
```

Reply:
I was thinking about it too for a while.  

What about:

    let $$ = quiet(nothrow($))

Reply:
And store options in IIFE contexts? yep, it might work too.